src/Queue/Logger.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace App\Queue;
+
+use Psr\Log\LoggerInterface;
+use Symbiote\QueuedJobs\Services\QueuedJob;
+
+/**
+ * Class Logger
+ *
+ * This logger redirects all log messages to the queued job which makes the job data contain all relevant logs
+ *
+ * @package App\Queue
+ */
+class Logger implements LoggerInterface
```

Comment:
Would benefit from a config example. How are you injecting the logger only during queued job executions? Are you ensuring that a plaintext formatter is attached to it?

Reply:
Please see [this example](https://github.com/silverstripe/silverstripe-assets/pull/397/files#diff-87735b788335fa4c95bc9c07846c7f48R33). The logger is attached to a helper which is executed within the job.